SpEC/com_motion.py

```diff
@@ -208,10 +208,8 @@ def remove_avg_com_motion(path_to_waveform_h5='rhOverM_Asymptotic_GeometricUnits
     # Read the waveform data in
     w_m = read_from_h5(path_to_waveform_h5)
 
-    # Insert '_CoM' into the original `path_to_waveform_h5`; this will be the output path
-    path_to_new_waveform_h5 = path_to_waveform_h5.split('.h5', 1)
-    path_to_new_waveform_h5.insert(1, '_CoM.h5')
```

Comment:
I don't want it to error out, because it should still work if someone uses such a path; I just want to change the first occurrence of the `.h5` string.  (But for future reference, I just use standard exceptions like `raise ValueError("Some message")`.)